src/main/resources/swagger.api/schemas/sourceableField.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
I'm not 100% sold on the name of this class. Any better ideas?

Reply:
fieldWithConstrainedValues? discreteValueField (I definitely dislike this one the most, since literally every column technically has discrete values)? providedValueField?

Reply:
hmm, i'd maybe lean towards `valueField` with that, but that'd imply fields without values?